.changeset/old-dogs-sin.md

```diff
@@ -0,0 +1,8 @@
+---
+"@sit-onyx/nuxt": patch
```

Comment:
```suggestion
"@sit-onyx/nuxt": major
```

This would be a major / breaking change because the user has to change something in his config

Reply:
If you really want this, sure, but as long as we're in alpha mode, it should not create a "real" major release anyway right?